[Docs] Local development (#2439)
nuclio/nuclio: docs/devel/contributing.md

3. Enable go modules **GoLand > Preferences > Go > Go Modules (vgo)** and ensure `Enable Go Modules` box is checked
4. Run configurations are (currently) available for tests only

> For testing your changes locally, see [Running Nuclio on your computer with local changes](/docs/devel/test-nuclio-locally.md).

Reviewer: it is less "testing" but rather "developing"

> For testing your changes locally, see [Running Nuclio on your computer with local changes](/docs/devel/test-nuclio-locally.md).

Reviewer: ```suggestion
For testing your changes locally, see [Local development](/docs/devel/test-nuclio-locally.md).
```
